src/extpipes/__main__.py

```diff
@@ -63,63 +53,63 @@
 @click.version_option(prog_name="extpipes_cli", version=__version__)
 @click.option(
     "--cdf-project-name",
-    help="CDF Project to interact with the CDF API, the 'BOOTSTRAP_CDF_PROJECT',"
```

Comment:
ofc the `BOOTSTRAP_` references were a copy/paste error

Reply:
:)